release.md

```diff
@@ -0,0 +1,33 @@
+**Design system - build and deploy HOWTO**
```

Comment:
I think this file should be named `RELEASE.MD` instead of `release.md` to follow a general pattern.

Additionally, I'd suggest linking to this file from `README.md`.

Reply:
👌🏼

Reply:
👌🏼 linked and renamed